From 25dad0fae7fe1c932bb81bdb7a70838de57ca07a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zolt=C3=A1n=20B=C3=B6sz=C3=B6rm=C3=A9nyi?=
 <zboszor@gmail.com>
Date: Wed, 4 Jun 2025 12:40:28 +0200
Subject: [PATCH] Revert "Update ``license*`` metadata and include third-party
 licenses (#729)"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This reverts commit 4f453e565e2aa8ebc71616a7aadf0da46fed9a67.

Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>
Upstream-Status: Inappropriate [Yocto specific]
---
 pyproject.toml | 11 ++++-------
 1 file changed, 4 insertions(+), 7 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index e7d8ff6..7ede5d9 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [build-system]
 requires = [
-    "setuptools>=77",
+    "setuptools>=64",
     "setuptools-scm[toml]>=6.2",
     "Cython",
     "py-cpuinfo",
@@ -24,6 +24,7 @@ classifiers = [
     "Intended Audience :: Developers",
     "Intended Audience :: Information Technology",
     "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: MIT License",
     "Programming Language :: Python",
     "Topic :: Software Development :: Libraries :: Python Modules",
     "Operating System :: Unix",
@@ -33,12 +34,7 @@ classifiers = [
 maintainers = [
     { name = "Alistair Miles", email = "alimanfoo@googlemail.com" },
 ]
-license = "MIT"
-license-files = [
-    "LICENSE.txt",
-    "c-blosc/LICENSE.txt",
-    "c-blosc/LICENSES/*",
-]
+license = { text = "MIT" }
 
 [project.urls]
 "Bug Tracker" = "https://github.com/zarr-developers/numcodecs/issues"
@@ -98,6 +94,7 @@ crc32c = [
 "numcodecs.zfpy" = "numcodecs.zarr3:ZFPY"
 
 [tool.setuptools]
+license-files = ["LICENSE.txt"]
 package-dir = {"" = "."}
 packages = ["numcodecs", "numcodecs.tests"]
 zip-safe = false
-- 
2.49.0

